src/main/java/core/basesyntax/OperationTable.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+public class OperationTable {
```

Comment:
nice oop aproach, але враховуючи, що ми всеодно впираємось в конкретні операції buy / supply і додаткової гнучкості коду це не дає,  я б це спростив після зчитування файла до такого до такого: 
```suggestion
                String[] lineElements = line.split(CSV_SEPARATOR);
                if (lineElements[OPERATION_INDEX].equals(BUY_OPERATION)) {
                    buyAmount += Integer.parseInt(lineElements[AMOUNT_INDEX]);
                } else {
                    supplyAmount += Integer.parseInt(lineElements[AMOUNT_INDEX]);
                }
```
P.S. 
it's up to you, можеш залишити поточний підхід